test/fixtures/template_config.yml

```diff
@@ -0,0 +1,4 @@
+---
+vars:
+  containers: [ {'name': 'project-name-web'} ]
```

Comment:
Using `project-name-web` here is confusing to me because it is inconsistent-- in the above `fixtures/kubernetes-config.yml` template does not use the `{{}}` brackets for the metadata fields where this text is used on #5 & 19.

Reply:
Ah ok I just added this as an example of the flexibility we have in terms of using a for loop over a list.  What would be a better variable name here?